2017/AUG.md

```diff
@@ -0,0 +1,31 @@
+## Index Of Dishes for AUGUST 2017
+
+#### All problems are open to contributions, but prefer working and improving solutions marked with [![image](../img/WA.png)](#) and [![image](../img/TLE.png)](#). Simply [Create an Issue](https://github.com/aashutoshrathi/CodeChef/issues/new) and Send a PR when assigned.
+
+##### Here in this Table ![image](../img/CC.png) links to solution on CodeChef and ![image](../img/GH.png) links to solution on this repository.
+
+
+| Contest | Problem | [ECR](#ecr) | C | C++ | Java | Python |
+|:--------------|:----------------:|:----------------:|:----------------:|:----------------:|:-----------------:|:-----------------:|
+<a name="long"></a>
```

Comment:
This Table style is broken one, fix it in both files. Update your master and check monthwise files.